### ISSUE TITLE
Setting width, scaling and differences in item size?

### ISSUE THREAD

**Issue by michelh77** [Jaliborc/Bagnonium#35]
I'm a new Combuctor user and new to Github as well, so please excuse me if my suggestions have already been mentioned.

First of all: I love the idea behind Combuctor, finding stuff is so fast! It looks good as well. But I'm also a neat freak, and I currently use BaudBag to display my bags with a width of 16 items so that a complete gear set can be put in one row. What would really make Combuctor better for me is if I was able to do this also with Combuctor. Also, I make my bags quite small usually, so adjusting the size of the items would be awesome as well.

Speaking of size: when I filter or search the search results are shown bigger. Why is that? It feels strange.

TL:DR version:
- Love the addon, finding stuff is fast!
- Please consider adding setting width in # of items
- Please consider adding setting size of items in % of original
- Always showing items/results in the same size makes more sense to me
